src/constants.ts

```diff
@@ -77,3 +77,5 @@ export const AircraftTypeList = {
     a32nx: '32:1044695612337168494',
     a380x: '38:1044695718348210177',
 };
+
+export const PermissionsEmbedDelay = 10;
```

Comment:
```suggestion
export const PermissionsEmbedDelay = 10;

```
Should always have a newline/linefeed at the end of a file.